apply_manual_codes/apply_manual_codes.py

```diff
@@ -0,0 +1,135 @@
+                        }},
+                        Metadata(user, Metadata.get_call_location(), time.time())
+                    )
+                else:
+                    for label in labels:
+                        td.append_data(
+                            {key_of_coded: label},
+                            Metadata(label["Origin"]["OriginID"], Metadata.get_call_location(),
+                                     isoparse(label["DateTimeUTC"]).timestamp())
+)
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Merges manually cleaned files back into a traced data file.")
+    parser.add_argument("user", help="User launching this program, for use by TracedData Metadata")
+    parser.add_argument("json_input_path", metavar="json-input-path",
+                        help="Path to JSON input file, which contains a list of TracedData objects")
+    parser.add_argument("coded_input_path", metavar="coded-input-path",
+                        help="Directory to read manually-coded Coda files from")
+    parser.add_argument("json_output_path", metavar="json-output-path",
+                        help="Path to a JSON file to write merged results to")
+    parser.add_argument("scheme_input_path", metavar="scheme-input-path",
+                        help="Directory to read Coda scheme files from")
+                        
+    
+    args = parser.parse_args()
+    user = args.user
+    json_input_path = args.json_input_path
+    coded_input_path = args.coded_input_path
+    json_output_path = args.json_output_path
+    scheme_input_path = args.scheme_input_path
+
+    class MergePlan:
+        def __init__(self, raw_field, coda_name, coded_name):
+            self.raw_field = raw_field
+            self.coda_name = coda_name
+            self.coded_name = coded_name
+
+    merge_plan = [
+        MergePlan("Gender (Text) - mcf_demog", "Gender", "Gender_Coded"),
+        MergePlan("Location (Text) - mcf_demog", "Location", "Location_Coded"),
+        MergePlan("Education (Text) - mcf_demog", "Education", "Education_Coded"),
+        MergePlan("Training (Text) - mcf_demog", "Training", "Training_Coded"),
+        MergePlan("Work (Text) - mcf_demog", "Work", "Work_Coded"),
+        MergePlan("Age (Text) - mcf_demog", "Age", "Age_Coded"),
+    ]
+
+    # Load data from JSON file
+    with open(json_input_path, "r") as f:
+        data = TracedDataJsonIO.import_json_to_traced_data_iterable(f)
+
+    # Merge manually coded survey Coda files into the cleaned dataset
+    for plan in merge_plan:
+        coda_file_path = path.join(coded_input_path, "{}_coded.json".format(plan.coda_name))
+
+        if not path.exists(coda_file_path):
+            print("Warning: No Coda file found for key '{}'".format(plan.coda_name))
+            for td in data:
+                td.append_data(
+                    {plan.coded_name: None},
```

Comment:
I think this should actually be NOT_REVIEWED.

(It's still None in REACH because we introduced NOT_REVIEWED after creating all the input coda files, so this line was never executed after we added NOT_REVIEWED)